projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=harbor
+REPO_OWNER=goharbor
+REPO_SUBPATH=./src
+
+BASE_IMAGE_NAME?=eks-distro-base
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
```

Comment:
I believe this can go

Reply:
oh yeah, BASE_IMAGE_NAME can go as it is default to that. I don't think REPO_SUBPATH and DOCKERFILE_FOLDER can go. Am I missing something?

Reply:
What other images are lookign to build in this project?  Unless we are adding them now, I'd prefer we treat this as standard as possible